packages/psss/src/containers/Panels/OutbreaksPanel.js

```diff
@@ -25,9 +25,10 @@ import {
 } from '../../components';
 import { NotesTab } from '../NotesTab';
 import * as COLORS from '../../constants/colors';
-import { countryFlagImage } from '../../utils';
+import { countryFlagImage, getCountryName } from '../../utils';
 import { connectApi } from '../../api';
 import { useFetch } from '../../hooks';
+import { useParams } from 'react-router-dom';
```

Comment:
Unused?

Reply:
no it is used below. but it is in the wrong order in the imports list so i have moved it up

Reply:
Oh right! Cool, thanks. Do you get the red squiggles on webstorm?